src/app/builders/components/TableColumnDropdown.tsx

```diff
@@ -1,10 +1,40 @@
-import { FC } from 'react'
-import { CommonComponentProps } from '@/components/commonProps'
-import { Button } from '@/components/Button'
-import { MoreIcon } from '@/components/Icons/MoreIcon'
 import { MultipleSelectDropdown, SelectorOption } from '@/app/builders/components/MultipleSelectDropdown'
+import { Button } from '@/components/Button'
+import { CommonComponentProps } from '@/components/commonProps'
 import { CloseIcon } from '@/components/Icons/CloseIcon'
-import { useTableContext, useTableActionsContext } from '@/shared/context/TableContext'
+import { MoreIcon } from '@/components/Icons/MoreIcon'
+import { useTableActionsContext, useTableContext } from '@/shared/context/TableContext'
+import { FC } from 'react'
+import { ColumnId } from './Table/BuilderHeaderRow'
+
+type ColumnLabel = {
+  label: string
+  sublabel?: string
+}
+type DropdowColumn = Exclude<ColumnId, 'actions'>
+
+const LABELS: Record<DropdowColumn, ColumnLabel> = {
+  builder: {
+    label: 'Builder',
+  },
+  backer_rewards: {
+    label: 'Backer Rewards %',
```

Comment:
According to the latest design, we should have a sublabel here `current % - change`